proposal.md

```diff
@@ -0,0 +1,5 @@
+# Project Gutenberg In a Box Proposal
+
+We are Dylan DiGeronimo (ddigeron@stevens.edu), Ryan Locke (rlocke@stevens.edu), Naseem Miah (nmiah@stevens.edu), and Taleen Mitchell (tmitche2@stevens.edu), and we are extremely excited to have the chance to work with the Free Ebook Foundation to bring Project Gutenberg to more and more people around the world. The five of us posess a broad skillset, including experience with Linux systems, front and back end web development, database configuration, interface design, technical writing and documentation, and cybersecurity.
```

Comment:
I count 4 names, who's the fifth?

Reply:
We had a fifth member arranged (Rachel Stern, who I believe was reaching out to you at some point), but as she is a Cybersecurity major she requires departmental approval of the specific project, so she asked us to not put her on any proposals until she is sure of what she's doing.